sklearn/manifold/_mds.py

```diff
@@ -70,9 +70,8 @@ def _smacof_single(
         See :term:`Glossary <random_state>`.
 
     normalized_stress : bool, default=False
-        Whether use and return normed stress value (Stress-1) instead of raw
-        stress calculated by default. Only supported in non-metric MDS. The
-        caller must ensure that if `normalized_stress=True` then `metric=False`
+        Whether use and return normalized stress value (Stress-1) instead of raw
+        stress.
 
         .. versionadded:: 1.2
```

Comment:
Please add a `.. versionchanged:: 1.7` entry to specify that `normalized_stress=True` is now also supported when `metric=False`.

Reply:
Thanks, done.